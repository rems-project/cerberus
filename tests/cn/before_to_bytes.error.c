int main()
{
    int x = 0;
    int *p = &x;
    char *p_char = (char *)p;
    p_char[2] = 0xff;
}
