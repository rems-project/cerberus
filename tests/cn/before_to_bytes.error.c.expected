return code: 1
[1/1]: main -- fail
tests/cn/before_to_bytes.error.c:6:5: error: Missing resource for writing
    p_char[2] = 0xff;
    ~~~~~~~~~~^~~~~~ 
Resource needed: Block<char>(&&x[(u64)2'i32])
State file: file:///tmp/state__before_to_bytes.error.c__main.html
